config/300-sks.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: sks.networking.internal.knative.dev
+  labels:
+    serving.knative.dev/release: devel
+spec:
+  group: networking.internal.knative.dev
+  version: v1alpha1
+  names:
+    kind: ServerlessService
+    plural: sks
+    singular: sks
```

Comment:
I'd guess that you are going to break assumptions by not having singular and plural be `serverlessservice[s]`.

Reply:
Done